### ISSUE TITLE
Web Serial api missing from data-2.0.json

### ISSUE THREAD

**Issue by sachaw** [Fyrd/caniuse#5874]
It seems to have been missed, support was added in https://github.com/Fyrd/caniuse/pull/5816
but the data is not present in https://github.com/Fyrd/caniuse/blob/main/fulldata-json/data-2.0.json

**Comment by Fyrd** [comment]
Correct, the support data is still pending review, as such its "shown" value is "false" and it does not yet appear in the data files.

**Comment by sachaw** [comment]
Ah, thanks for the update, is there some tracker/timeline for this process? happy to contribute if work needs to be done to get this moved forward.

**Comment by Fyrd** [comment]
There is not, but if you'd like to look over the data and verify it yourself that can be helpful. Looks like it's nothing too complicated so I can probably get it in in the next few days.
